boskos/cmd/janitor/gcp/compute_cleaner.go

```diff
@@ -0,0 +1,73 @@
+package compute_cleaner
+
+import (
+	"fmt"
+	"time"
+
+	"golang.org/x/net/context"
+	"google.golang.org/api/compute/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type ComputeCleaner struct {
+	project        string
+	zone           string
+	computeService *compute.Service
+	ctx            context.Context
+	opName         string
+}
+
+func NewCleaner(project, zone string, computeService *compute.Service, ctx context.Context) (*ComputeCleaner, error) {
```

Comment:
Typically the janitor cleans up all zones/regions. Does it make sense to have a single zone provided here? Should this be a list of zones? Or maybe it should automatically query for the list of all known zones?